app/i18n/locales/en-US.json

```diff
@@ -226,5 +229,39 @@
+  "wallet.trezor.dialog.trezor.step.about.introText.line.2": "It is more secure because your private key never leaves the hardware wallet.",
+  "wallet.trezor.dialog.trezor.step.about.introText.line.3": "This protects your funds even if your computer is compromised due to malware, phishing attempts, etc.",
+  "wallet.trezor.dialog.trezor.step.about.label": "ABOUT",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.1.part1": "Only Supports ",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.1.part2.link": "https://github.com/trezor/trezor-core/blob/be58549fd9fe53df237a7318f754b5ec23975425/ChangeLog#L12",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.1.part2.link.text": "Trezor Model T with version 2.0.8",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.1.part3": " or later.",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.2": "The Trezor device must be pre-initialized.",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.3": "The Trezor device screen must be unlocked.",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.4": "Only one Trezor device can be connected to the computer at any time.",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.5": "Your computer must remain connected to the Internet throughout the process.",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.6": "The Trezor device must remain connected to your computer throughout the process.",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.header": "Prerequisites",
+  "wallet.trezor.dialog.trezor.step.connect.error.101": "Falied to connect trezor.io. Please check your Internet connection and retry.",
+  "wallet.trezor.dialog.trezor.step.connect.error.102": "Necessary permissions were not granted by the user. Please retry.",
+  "wallet.trezor.dialog.trezor.step.connect.error.103": "Cancelled. Please retry.",
+  "wallet.trezor.dialog.trezor.step.connect.error.999": "Something unexpected happened. Please retry.",
+  "wallet.trezor.dialog.trezor.step.connect.introText.line.1": "After connecting your Trezor device to your computer, press the Connect button.",
+  "wallet.trezor.dialog.trezor.step.connect.introText.line.2": "A new tab will appear. Please follow the instructions in the new tab.",
+  "wallet.trezor.dialog.trezor.step.connect.introText.line.3": "This process shares the Cardano public key with Yoroi.",
+  "wallet.trezor.dialog.trezor.step.connect.label": "CONNECT",
+  "wallet.trezor.dialog.trezor.step.connect.liveMessage.checkingTrezorDevice": "Checking the Trezor device. Please follow the instructions in the new tab...",
+  "wallet.trezor.dialog.trezor.step.save.error.101": "Falied to save. Please check your Internet connection and retry.",
+  "wallet.trezor.dialog.trezor.step.save.label": "SAVE",
+  "wallet.trezor.dialog.trezor.step.save.walletName.hint": "!!!Enter wallet name",
```

Comment:
get rid of `!!!`

Reply:
Thanks, it was a bug, this property was not used at all so did not come to focus.

Reply:
DONE